### DISCUSSION TITLE
Formatting issues with list(ordered/unordered)...

### DISCUSSION THREAD

**Discussion by zhang-stephen** [TriliumNext/Trilium#3229] [Q&A]
I am not sure this issue is caused by CKEditor or the other module...

### Cannot attach a block to item in ordered/unordered list
The markdown allows block elements to be attached to existed item in list, by indents.

```markdown
+ Forward-compatibility
    > Forward compatibility in NR is achieved by self-contained and well-confined transmissions. Self-containment means that data in a slot and a beam is decodable without dependency on other slots and beams. Well-confined transmissions refer to keeping transmissions confined in frequency and time domains to allow future inclusion of new types of transmissions in parallel with legacy transmissions.
``` 

after rendered:
![](https://user-images.githubusercontent.com/17326718/197402970-f9986b57-caf8-4826-95dd-9c0c0cfc0162.png)

But in trilium, I got this, which looks weird. The button `increase indent` in `edit block` is effective to contents in the block, instead of the block itself.
![image](https://user-images.githubusercontent.com/17326718/197402798-668eca18-0663-4442-9346-d35988e4a945.png)

How to make the trilium behaves like the markdown renderer?

### the item number is broken if trying to attach block to item...

like this screenshot, notice the third item:
![image](https://user-images.githubusercontent.com/17326718/197403393-d3485345-1c83-48e5-a54e-51c9831355f8.png)

I think this is caused by the first issue, the editor does not know there are previous item number before the citation block...

Anyone could help on these issues?

**Comment by zadam** [comment]
CKEditor does not support block elements in lists. IIRC they are working on it, but it's surprisingly complex issue.
